connector/spanmetricsconnector/README.md

```diff
@@ -14,38 +14,29 @@
 
 ## Overview
 
-Aggregates Request, Error and Duration (R.E.D) metrics from span data.
+Aggregates Request, Error and Duration (R.E.D) OpenTelemetry metrics from span data.
 
 **Request** counts are computed as the number of spans seen per unique set of dimensions, including Errors.
-For example, the following metric shows 142 calls:
-```
-calls_total{http_method="GET",http_status_code="200",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET"} 142
-```
 Multiple metrics can be aggregated if, for instance, a user wishes to view call counts just on `service_name` and `operation`.
 
-**Error** counts are computed from the Request counts which have an "Error" Status Code metric dimension.
-For example, the following metric indicates 220 errors:
-```
-calls_total{http_method="GET",http_status_code="503",operation="/checkout",service_name="frontend",span_kind="SPAN_KIND_CLIENT",status_code="STATUS_CODE_ERROR"} 220
-```
+**Error** counts are computed from the Request counts which have an `Error` Status Code metric dimension.
 
 **Duration** is computed from the difference between the span start and end times and inserted into the
 relevant latency histogram time bucket for each unique set dimensions.
-For example, the following latency buckets indicate the vast majority of spans (9K) have a 100ms latency:
-```
-latency_bucket{http_method="GET",http_status_code="200",label1="value1",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET",le="2"} 327
-latency_bucket{http_method="GET",http_status_code="200",label1="value1",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET",le="6"} 751
-latency_bucket{http_method="GET",http_status_code="200",label1="value1",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET",le="10"} 1195
-latency_bucket{http_method="GET",http_status_code="200",label1="value1",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET",le="100"} 10180
-latency_bucket{http_method="GET",http_status_code="200",label1="value1",operation="/Address",service_name="shippingservice",span_kind="SPAN_KIND_SERVER",status_code="STATUS_CODE_UNSET",le="250"} 10180
-...
-```
 
-Each metric will have _at least_ the following dimensions because they are common across all spans:
-- Service name
-- Operation
-- Span kind
-- Status code
+
+Each metric will have _at least_ the following dimensions because they are common
+across all spans:
+
+- `span.name`
+- `operation`
```

Comment:
I guess this will be updated once you've rebased with `main` to pickup changes from https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18746.